static/lang/re-en.json

```diff
@@ -1930,6 +1934,9 @@
                         "SecretLabel": "Moderate Eagle Eye Elixir (To Find Secret Doors or Traps)"
                     }
                 },
+                "FightersFork": {
+                    "FluidLengthLabel": "Fighter's Fork - Fluid Length"
```

Comment:
```suggestion
                    "FluidLengthLabel": "Fighter's Fork — Fluid Length"
```
I guess the nice thing about localizing all these is we can revisit them, the new activation format is going to take some thinking about how we want to go about localizing stuff to not balloon re-en.json.